svelte.config.js

```diff
@@ -24,6 +24,9 @@ const config = {
                     $lib: path.resolve('./src/lib'),
                 },
             },
+            ssr: {
```

Comment:
Can you explain me this?

Reply:
This line of code was added in order to prevent the error which was thrown while running the app. The error was saying "Cannot use import statement outside a module" which was connected to popper.js script in node modules. After investigation on the internet I found the solution from 24th of May  [Cannot use import statement outside a module #4504](https://github.com/sveltejs/kit/issues/4504) where it was saying that this issue is related to the bug with @popperjs/core